### DISCUSSION TITLE
No syntax highlighting in markdown files

### DISCUSSION THREAD

**Discussion by eguiraud** [helix-editor/helix#9252] [Troubleshooting] [answered]
Hello, I cannot get syntax highlighting for markdown files to work in Helix 23.10 although as far as I can tell from https://github.com/helix-editor/helix/issues/399 and https://github.com/helix-editor/helix/issues/215 it should work.

This is what I see when I open a markdown file. It is correctly recognized as markdown and tree-sitter features like go-to-references or go-to-symbol work, but no colors/highlighting of any sort. I'm on Arch Linux. I couldn't spot anything suspicious in the logs.

```
~ helix --health markdown
Configured language servers:
  ✓ marksman: /usr/bin/marksman
Configured debug adapter: None
Highlight queries: ✓
Textobject queries: ✘
Indent queries: ✘
```

![image](https://github.com/helix-editor/helix/assets/10999034/1d25fb2b-a314-422d-8ace-04b744cd9f91)

**Comment by the-mikedavis** [comment, marked as answer]
Is that the nord theme? This may be https://github.com/helix-editor/helix/issues/9131 which was fixed on master but isn't released yet

**Reply by eguiraud** [reply]
Wow, thank you, that's exactly it! It did not occur to me that the theme might disable syntax highlighting features.

**Reply by TornaxO7** [reply]
@eguiraud pls don't forget to mark your question as answered

**Reply by eguiraud** [reply]
oops, sorry about that!
